Makefile

```diff
@@ -55,8 +65,18 @@ vet: ## Run go vet against code.
 	go vet ./...
 
 .PHONY: lint
-lint: ## Run golangci-lint against code.
-	golangci-lint run ./...
+lint: ## Run golint against code.
+	golint ./...
```

Comment:
Any reason to change this?

Reply:
Container running it might now have golangci-lint and to keep consistent with `code/check` 🤔 . Actually, since `golint` archived probably better to use the `golangci-lint` everywhere and ensure prow know what is that

Reply:
golangci-lint afaik, is the preferred linter these days, so personally I'd stick with it. Also there is a config in this repo for it, that is currently getting used both locally and by the GH action that we are using to run it (leaves comments on the PRs when linters stuff fails). so you'd need to change that if you were replacing it.

RE. enabling openshift CI, do we see us moving all our CI tasks to there or are we planning to keep GH actions? I see a lot of advantages in enabling prow for better management of changes getting back to main (tide managing merges etc..) , but we actually don't have to run all the tests there unless we really want to.